### PR TITLE
Update react-native peer dependency and add onPress callback

### PR DESCRIPTION
**Pull request by alexrabarts**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -27,7 +27,7 @@
     "react-tween-state": "0.0.5"
   },
   "peerDependencies": {
-    "react-native": "^0.4.0"
+    "react-native": ">=0.4"
```
Comment (naoufal) [inline package.json:30]: @alexrabarts I'm surprised `^0.4.0` didn't work.  The expected behaviour is to bump minor versions when needed, which should cover `0.5.0` and `0.6.0`.

**Comment by alexrabarts** [comment]
You can test the semver here: http://semver.npmjs.com/

That suggests `^0.4.0` won't work with `0.5.0` and `0.6.0`.

**Comment by naoufal** [comment]
@alexrabarts Sweet. Didn't know npm had a tool for testing semver.

**Comment by naoufal** [comment]
Thanks for your contribution.  Merging this. :+1:

**Comment by naoufal** [comment]
@alexrabarts Just updated the package on npm.

**Comment by alexrabarts** [comment]
Thank you!

**Pull request merged**